pkgs/development/libraries/sundials/default.nix

```
@@ -5,15 +5,20 @@
 , blas
 , lapack
 , gfortran
-, lapackSupport ? true }:
+, suitesparse
+, lapackSupport ? true
+, kluSupport ? true }:
 
 assert (!blas.isILP64) && (!lapack.isILP64);
 
 stdenv.mkDerivation rec {
   pname = "sundials";
   version = "5.3.0";
 
-  buildInputs = [ python ] ++ stdenv.lib.optionals (lapackSupport) [ gfortran blas lapack ];
+  buildInputs =
+    [ python ]
+    ++ stdenv.lib.optionals (lapackSupport) [ gfortran blas lapack ]
+    ++ stdenv.lib.optionals (kluSupport) [ suitesparse ];
```

most common formatting I see, and I find it much more readable
```suggestion
  buildInputs =[
    python
  ] ++ stdenv.lib.optionals (lapackSupport) [
    gfortran
    blas
    lapack
  ] ++ stdenv.lib.optionals (kluSupport) [
    suitesparse
  ];
```